server/adapters/clickhouse.go

```diff
@@ -612,6 +646,42 @@ func (ch *ClickHouse) reformatValue(v interface{}) interface{} {
 	return v
 }
 
+func (ch *ClickHouse) ValidateWritePermission() error {
```

Comment:
After moving test eventContext creation outside storage, this func can be omitted.

Reply:
Function was removed